deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -47,21 +50,37 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+          ports:
+            - name: healthz
+              containerPort: 9910
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+            periodSeconds: 2
+            failureThreshold: 5
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.3.0
+          image: public.ecr.aws/eks-distro/kubernetes-csi/external-provisioner:v2.1.1-eks-1-18-2
           args:
-            - --timeout=5m
             - --csi-address=$(ADDRESS)
-            - --v=5
-            - --enable-leader-election
```

Comment:
I see that the image is changing, just wanted to make sure dropping the leader election flags is intended.

Reply:
no, I added back